README.md

```diff
@@ -27,7 +27,7 @@ Library does not have other dependencies than fs2 and zookeeper client itself:
 
 version  |    scala  |   fs2  |  zookeeper     
 ---------|-----------|--------|--------- 
-0.4.0    | 2.11, 2.12| 1.0.0  | 3.4.10   
```

Comment:
Could you change this back? I will add this line once I will release the version, just so that we do not confuse people.

Reply:
done :)